Runtime/Client/LootLockerStateData.cs

```diff
@@ -195,6 +195,9 @@ public static LootLockerPlayerData GetStateForPlayerOrDefaultStateOrEmpty(string
             }
             string playerULIDToGetDataFor = string.IsNullOrEmpty(playerULID) ? ActiveMetaData.DefaultPlayer : playerULID;
 
+            // Make this player the default for requests if there is no default yet or if the current default is not currently active
+            bool shouldBeMadeDefault = !playerULID.Equals(ActiveMetaData.DefaultPlayer, StringComparison.OrdinalIgnoreCase) && ActivePlayerData.Count == 0;
```

Comment:
Flip the check to do the least costly first
```suggestion
            bool shouldBeMadeDefault = ActivePlayerData.Count == 0 && !playerULID.Equals(ActiveMetaData.DefaultPlayer, StringComparison.OrdinalIgnoreCase);
```